### PR TITLE
Generate file ID with id.GenerateID()

### PR DESCRIPTION
**Pull request by kkaneda**
Verified that the ID has the `file-` prefix in OpenAI.

### CODE REVIEW

**Comment by kkaneda** [comment]
@Ladicle , does this look good to you?

**Comment by Ladicle** [comment]
Looks good to me, s3 name policy allows Alphanumeric (`0-9 a-z A-Z`) and  symbols (`! - _ . * ' ( )`). Just to confirm, do we also align with the random string length (OpenAI is 24)?

**Review by Ladicle — APPROVED** [review]

**Comment by kkaneda** [comment]
Thanks for checking! You're right. I somehow miscounted. :(

image

File: server/internal/server/files.go [hunk 1]
```diff
@@ -65,7 +65,12 @@ func (s *S) CreateFile(
 	}()
 
 	log.Printf("Uploading the file to S3\n")
-	fileID := newFileID()
+	fileID, err := id.GenerateID("file-", 20)
```
Comment (kkaneda) [inline server/internal/server/files.go:68]:
```suggestion
	fileID, err := id.GenerateID("file-", 24)
```

**Pull request merged**